Docker/Dockerfile

```diff
@@ -11,7 +11,7 @@ WORKDIR "$WORKDIR/narwhal"
 ARG BUILD_MODE
 
 # Install basic dependencies
-RUN apt-get -qq update \
+RUN DEBIAN_FRONTEND=noninteractive apt-get -qq update \
```

Comment:
If the `-qq` doesn't work do we still want to leave it there?

Reply:
For `apt update` it does work, for `install` it doesn't.